### PR TITLE
New topic for platform service migration

### PR DESCRIPTION
**Pull request by ShellyXueHan**
## Summary
Quick Summary Here

As platform service will be migrating several services, I'm starting this topic to capture the docs.

## Notable Changes 
- a new topic

### CODE REVIEW

File: app-web/topicRegistry/platform-service-migration.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Platform Service Migration",
+  "description": "A series of migration plan for the platform services provided",
+  "attributes": {
+    "labels": [
```
Comment (patricksimonian) [inline app-web/topicRegistry/platform-service-migration.json]: labels are key value pairs and not an array just like openshift.

File: app-web/topicRegistry/platform-service-migration.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Platform Service Migration",
+  "description": "A series of migration plan for the platform services provided",
```
Comment (patricksimonian) [inline app-web/topicRegistry/platform-service-migration.json]: A series of plans and guidances for  migrating many of the  platform  services that are provided to application teams.

File: app-web/topicRegistry/platform-service-migration.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "sourceProperties": {
+    "sources": [
+      {
+        "sourceType": "github",
+        "sourceProperties": {
+          "url": "https://github.com/BCDevOps/developer-experience",
+          "owner": "BCDevOps",
+          "repo": "developer-experience",
+          "branch": "shelly-kc-migration",
+          "throttle": 1000,
```
Comment (patricksimonian) [inline app-web/topicRegistry/platform-service-migration.json]: remove throttle:P

File: app-web/topicRegistry/platform-service-migration.json (hunk 1 above)
Comment (ShellyXueHan) [reply app-web/topicRegistry/platform-service-migration.json]: not sure what you mean...

Comment (patricksimonian) [reply app-web/topicRegistry/platform-service-migration.json]:
```js
labels: {
  ministry: 'foo',
}
```

**Pull request merged**